docs/src/main/sphinx/security/certificate.rst

```diff
@@ -0,0 +1,108 @@
+==========================
+Certificate authentication
+==========================
+
+You can configure Trino to support client-provided certificates validated by the
+Trino server on initial connection.
+
+.. important::
+
+    This authentication method is only provided to support sites that have an
+    absolute requirement for client authentication *and already have* client
+    certificates for each client. Sites in this category have an existing PKI
+    infrastructure, possibly including an onsite Certificate Authority.
+
+    This feature is not appropriate for sites that would need to generate a set
+    of client certificates in order to use this authentication method. Consider
+    instead using another :doc:`authentication method
```

Comment:
should we link to this or to the section in the toctree or security overview? I kinda think the section in the security overview might be best..

Reply:
OK